upsun/15-symfony-flex-upsun.yaml

```diff
@@ -47,7 +48,7 @@ template: |
                         passthru: "/{{.FrontController}}"
 
             mounts:
-                "/var": { source: storage, source_path: var }
+                "/var/cache": { source: instance, source_path: var/cache }
```

Comment:
For thoughts: var/cache/pool contains parts that would be better in a `storage` (app pools) and others which are better in `instance` (system pools).
We might check how to make a cleaner separation in Symfony itself.

Reply:
PR updated, see the line below this one and https://github.com/symfonycorp/platformsh-tools/pull/44